pkg/skaffold/schema/validation/validation.go

```diff
@@ -189,6 +189,11 @@ func validateDockerNetworkMode(artifacts []*latest.Artifact) (errs []error) {
 		if mode == "none" || mode == "bridge" || mode == "host" {
 			continue
 		}
+		containerRegExp := regexp.MustCompile("^container:[a-zA-Z0-9][a-zA-Z0-9_.-]*$")
```

Comment:
Is there a way to validate if the containerName or Id is valid?

Reply:
AFAIK, this `validation` process prevents creating the `runContext`. In order to check whether the container exists, we would need this `runContext` (not created yet) injected in the `validation`, not possible with the current validation design. 

We might need to extract validating the `container:<id|name>` to a post-validation depending on the `runContext`, but that sounds to me like breaking the current workflow. I believe delegating the container validation to docker should be enough, since it explicitly shows that the container is missing.

In case this validation is a MUST, we could create a new issue and address it. WDYT?

Reply:
Sorry @tejal29 , I asked you for a new review by mistake.

Reply:
I added a new commit with the validation of the container